### PR TITLE
Rajshri All Assi 1 to 6

### PR DESCRIPTION
**Pull request by Rajashree07**

### CODE REVIEW

File: Assignments/HTML_CSS/Assign1/ass1q2.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+<title>Que 2:Write a HTML prog to get the follwing output</title>
+</head>
+<body>
+<h1> HTML</h1>
+<h2> Introduction</h2>
+<h3> HTML is a standard markup language for Web Page.</h3>
+<h2> HTML Editor</h2>
+<h3> Notepad or TexEditor.</h3>
+<h4><pre> Web pages can be created and modified by using professional HTML editors.
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/ass1q2.html]: for paragraph use p tag

File: Assignments/HTML_CSS/Assign1/ass1q4.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<html>
+<head>
+<title>
+SELF INTRODUCTION!
+</title>
+</head>
+<h2 style="background-color:yellow;">Self Introduction</h2>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign1/ass1q4.html:7]: use this in body tag

File: Assignments/HTML_CSS/Assign2/ass2.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<html>
+<head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/ass2.html]: head tag closing missing

File: Assignments/HTML_CSS/Assign2/updated_ass2.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+<body bgcolor="pink">
+</head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/updated_ass2.html:4]: wrong practice

File: Assignments/HTML_CSS/Assign2/updated_ass2.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	<hr>
+	<h2><b>Initial public offering</b></h2>
+	<br>Google's initial public offering (IPO) took place five years later, on 19 <sup>th</sup> August 2004.
+        <br>
+         At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for<del> 30 </del><ins>20</ins>
+         years, until the year 2024.
+	 <marquee behavior="alternate"direction="right"scrolldelay=5>
+         <p style="text-align:center;color:green;"><b>Thank You!!</b></p>
+	 
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/updated_ass2.html:27]: remove extra lines

File: Assignments/HTML_CSS/Assign3/A3_1.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+     </ul>
+      <li><a1> SYSTEM-2</li>
+     <ul><a1>
+      <li><a1>Triveini</li>
+      <li><a1>Kalyani</li>
+     </ul>
+     </ol>
+     </ol>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/A3_1.html:54]: what is use of a1 tag and a tag

File: Assignments/HTML_CSS/Assign3/A3_2.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+     <body style="background-color:blue">
+  
+        <h2 style="text-align: center;">Ambersoft</h2>
+        <p style="text-align: center;">
+        <a href="" target="_top"><ins> Home </ins></a>|
+        <a href="" target="_top"><ins> MI-MEAN </ins></a>|
+        <a href="" target="_top"><ins>  Contact </ins><br></a>
+        </p>
+          <img src="nature1.jpg" alt="no image" style="width: 300px;height: 300px; float: left;">
+	  <p><h3>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/A3_2.html]: how can you use h3 tag in p tag

File: Assignments/HTML_CSS/Assign3/RajMainA3_2.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+      </style>
+  </head>
+  <body style="background-color:rgb(27, 159, 211)">
+      <h2 style="text-align: center;">Ambersoft</h2>
+      <p style="text-align: center;">
+      <a href="RajMainA3_2.html" target="_top"><ins> Home </ins></a>|
+      <a href="Rajmi9A3_2.html" target="_top"> <ins> MI-MEAN </ins></a>|
+      <a href="A3_2.html" target="_top"><ins> Contact </ins></a>
+      <br>
+      <h3>Home Page</h3>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/RajMainA3_2.html:16]: same mistake h3 in p tag

File: Assignments/HTML_CSS/Assign3/RajMainA3_2.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        </p>
+        <h2 style="text-align: center;"></h2>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/RajMainA3_2.html:18]: missing content

File: Assignments/HTML_CSS/Assign3/RajMainA3_2.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+        <p>
+          <img class = "leftPic"  src="nature1.jpg" alt="no image" width="300px" height="300px" style="float: left;">
+          <img class = "rightPic" src="nature1.jpg " alt="no image" width="300px" height="300px" style="float: right">
+     <h3>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/RajMainA3_2.html]: p tag closing missing

File: Assignments/HTML_CSS/Assign3/Rajmi9A3_2.html [hunk 11]
```diff
@@ -0,0 +1,27 @@
+<html>
+    <html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/Rajmi9A3_2.html]: double html tag

File: Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html [hunk 12]
```diff
@@ -0,0 +1,51 @@
+<html>
+        <head>
+                <title> table tag</title>
+        <style>
+        table,th,td{
+        border:1px solid black;
+        !-- border-collapse: collapse;
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html:7]: remove this extra

File: Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+        </tr>
+	<tr>
+	<th >Females</th>
+        <td>1.7</td>
+        <td>0.002</td>
+        <td>43%</td>
+        </tr>
+        </table>
+        </body>
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html]: HTML closing missing

File: Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+        <head>
+                <title> table tag</title>
+        <style>
+        table,th,td{
+        border:1px solid black;
+        !-- border-collapse: collapse;
+        }
+	        table,th,td{
+        border:1px solid black;
+        !-- border-collapse: collapse;
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign4/RajashreeAss4_1.html:11]: also from here

File: Assignments/HTML_CSS/Assign4/RajashreeAss4_2.html [hunk 15]
```diff
@@ -0,0 +1,70 @@
+<html>
+        <head>
+                <title> marge table</title>
+        <style>
+        table,th,td{
+        border:1px solid black;
+        !-- border-collapse: collapse;
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign4/RajashreeAss4_2.html:7]: same mistake

File: Assignments/HTML_CSS/Assign4/RajashreeAss4_3.html [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+                <td rowspan="2" style="background-color:lightsteelblue;">5:00</td>
+		<td>Introduction</td>
+	</tr>
+
+        <tr>
+               <td align=center>Validity: DTD and Relax NG</td>
+       </tr>
+        <tr>
+               <td rowspan=3>Tuesday</td>
+               <td style="background-color: rgb(229,233,21);"> 8:00 a.m.</th>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign4/RajashreeAss4_3.html]: check closing

File: Assignments/HTML_CSS/Assign5/CV.html [hunk 17]
```diff
@@ -0,0 +1,103 @@
+<html>
+ <head>
+<link type="text/css" rel="stylesheet" href="file:///home/shree/HTML/CV.css"/>
+<title></title>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/CV.html]: title missing

File: Assignments/HTML_CSS/Assign5/CV.html [hunk 18]
```diff
@@ -0,0 +1,103 @@
+<html>
+ <head>
+<link type="text/css" rel="stylesheet" href="file:///home/shree/HTML/CV.css"/>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/CV.html]: css file missing and use relative path

File: Assignments/HTML_CSS/Assign5/CV.html [hunk 19]
```diff
@@ -0,0 +1,103 @@
...
+                </tr>
+                <tr>
+                    <td>B.Sc.(Computer Science)</td>
+                    <td>Pune University</td>
+                    <td>71.35%</td>
+                    <td>2019</td>
+                </tr>
+            </table>
+	    <b><u>Project Summary</b></u>
+            <p>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/CV.html]: closing missing

File: Assignments/HTML_CSS/Assign5/RajashreeAss5.html [hunk 20]
```diff
@@ -0,0 +1,31 @@
+<html>
+        </style>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/RajashreeAss5.html:2]: opening missing

File: Assignments/HTML_CSS/Assign5/RajashreeAss5.html [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+        </head>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/RajashreeAss5.html:3]: opening is missing

File: Assignments/HTML_CSS/Assign5/RajashreeAss5.html [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+    <body bgcolor="bisque">
+        <h2 class="format_font">Rajashree Patil</h2>
+        <div>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/RajashreeAss5.html:6]: closing missing

File: Assignments/HTML_CSS/Assign5/RajashreeAss5.html [hunk 23]
```diff
@@ -0,0 +1,31 @@
...
+            My Name is Rajashree Patil
+        <h3 class="format_font format head">
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/RajashreeAss5.html:8]: use p tag for paragraph

File: Assignments/HTML_CSS/Assign5/class.css [hunk 24]
```diff
@@ -0,0 +1,19 @@
+          <style>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/class.css]: wrong practice

File: Assignments/HTML_CSS/Assign5/class.css [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+             background-color:orange;
+              border-style: solid;
+              border-color:orange;
+          }
+          .format-head{
+              background-color: bisque;
+              border-style: solid;
+            
+          }
+          </style>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/class.css]: wrong practice

File: Assignments/HTML_CSS/Assign5/hobbies.html [hunk 26]
```diff
@@ -0,0 +1,16 @@
...
+            <h1><b>My Hobbies</b></h1>
+            <ol>
+                <li>Listen To Music.</li>
+                <li>Watch Movies.</li>
+                <li>Cooking</li>
+                <li>Learning HTML.</li>
+            </ol>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/hobbies.html:16]: add new line

File: Assignments/HTML_CSS/Assign5/hobbies.html [hunk 27]
```diff
@@ -0,0 +1,16 @@
+<html>
+    <head>
+        <title>My Hobbies</title>
+    </head>
+    <body bgcolor="lightgreen">
+        <div class="section">
+            <h1><b>My Hobbies</b></h1>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign5/hobbies.html]: no need of b tag

File: Assignments/HTML_CSS/Assign6/RajashreeAss6.html [hunk 28]
```diff
@@ -0,0 +1,92 @@
+<html>
+    <head>
+        <title>Form Ass6</title>
+    </head>
+    <link type="text/css" rel="stylesheet" href="file:///home/shree/HTML/form.css">
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/RajashreeAss6.html:5]: use relative path

File: Assignments/HTML_CSS/Assign6/RajashreeAss6.html [hunk 29]
```diff
@@ -0,0 +1,92 @@
...
+          <legend> Resume</legend>
+          <label for="myfile">Upload Resume:       </label>
+          <input type="file" id="myfile" name="myfile"><br><br>
+          <label for="comm">Comments</label>
+          <textarea id="comm" name="comm" placeholder="Comment here.." style="height:200px"></textarea>
+          </fieldset>
+          <input type="submit" value="Submit">
+        </form>
+        </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/RajashreeAss6.html:92]: add new line at end

File: Assignments/HTML_CSS/Assign6/form.css [hunk 30]
```diff
@@ -0,0 +1,13 @@
...
+  width: 50%;
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+}
+h2{
+    background-color:lightblue;
+    text-align: right;
+}
+</style>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign6/form.css]: wrong practice

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need indentation 
do correct all changes

**Review by pradip0852 — APPROVED** [review]

**Review by pradip0852 — APPROVED** [review]

**Review by Manishnerkar — APPROVED** [review]

**Pull request merged**